From cc3ff0af19499f0aaa587f42d3091b4dff49208f Mon Sep 17 00:00:00 2001
From: mulhern <amulhern@redhat.com>
Date: Mon, 27 Nov 2017 10:02:44 -0500
Subject: [PATCH] dm thin: fixes in thin-provisioning.txt
Git-commit: cc3ff0af19499f0aaa587f42d3091b4dff49208f
References: bsc#1111974
Patch-mainline: v4.16-rc1

Make the format string for thinpool status more correct.

Swap the order of two items to correspond with reality.

Signed-off-by: mulhern <amulhern@redhat.com>
Signed-off-by: Mike Snitzer <snitzer@redhat.com>
Signed-off-by: Coly Li <colyli@suse.de>

---
 Documentation/device-mapper/thin-provisioning.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/device-mapper/thin-provisioning.txt b/Documentation/device-mapper/thin-provisioning.txt
index 3852dade03e4..fb10d187090e 100644
--- a/Documentation/device-mapper/thin-provisioning.txt
+++ b/Documentation/device-mapper/thin-provisioning.txt
@@ -276,7 +276,7 @@ ii) Status
 
     <transaction id> <used metadata blocks>/<total metadata blocks>
     <used data blocks>/<total data blocks> <held metadata root>
-    [no_]discard_passdown ro|rw
+    ro|rw|out_of_data_space [no_]discard_passdown
 
     transaction id:
 	A 64-bit number used by userspace to help synchronise with metadata
-- 
2.19.0

